tm2/pkg/db/goleveldb/go_level_db_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/db/internal"
 	"github.com/gnolang/goleveldb/leveldb/opt"
 	"github.com/jaekwon/testify/assert"
-	"github.com/stretchr/testify/require"
+	"github.com/jaekwon/testify/require"
```

Comment:
Do  we need to modify this on this PR?

Reply:
Here I was making it consistent with the above import, not crucial but I think it doesn't hurt...

Reply:
We should have a discussion, outside this PR, on these packages

Reply:
afaik, we already had a discussion about this and we settled on using the jaekwon's one.

Ref: https://github.com/gnolang/gno/issues/777#issuecomment-1632014195